app/nasa_collaboration/page.module.css

```diff
@@ -2,11 +2,65 @@
   background-color: black;
   width: 100vw;
   height: 100vh;
+  color: white;
+  font-family: Arial, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
+.fullBGpicture {
```

Comment:
In CSS, the convention is to use [kebab case](https://developer.mozilla.org/en-US/docs/Glossary/Kebab_case) for class names.